src/nvim/menu.c

```diff
@@ -1144,10 +1145,10 @@ get_menu_cmd_modes (
  */
 static char_u *popup_mode_name(char_u *name, int idx)
 {
-  int len = (int)STRLEN(name);
+  size_t len = STRLEN(name);
 
   char_u *p = vim_strnsave(name, len + 1);
-  memmove(p + 6, p + 5, (size_t)(len - 4));
+  memmove(p + 6, p + 5, len - 4);
```

Comment:
Done. (The surrounding code ensures this function is only called when name starts with `PopUp`).